ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -298,14 +315,15 @@ def traffic_cb(self, traffic_msg):
         if traffic_msg.data != self.next_tl_wp:
             if self.dyn_test_stoplight is False:
                 self.next_tl_wp = traffic_msg.data
-                rospy.logwarn("new /traffic_waypoint message received at wp: %d."
+                rospy.logwarn("new /traffic_waypoint message received is wp: %d."
                           "while car is at wp %d", self.next_tl_wp,
                           self.final_waypoints_start_ptr)
             else:
                 rospy.logwarn("Ignoring /traffic_waypoint message while testing.")
```

Comment:
This outputs to console repeatedly while the test waypoint is active, maybe should move this message to where the dynamic config updates the value when received so it will only output once.